### PR TITLE
Adding Environment for Docker Timezones

### LINKED ISSUE #79: Change docker container timezones
Change Timezone in docker containers for local. It seems that the timezones in the php configuration and MySQL are not the same.

Port of Medology/stdcheck.com#1625.

### PR DESCRIPTION
**Pull request by TaysirTayyab**
Fixes #79.
Port of Medology/stdcheck.com#1702.

**Method**
`docker-compose` and `docker run` do not set timezones on containers meaning everyone has their own timezone.

**Solution:**
New environment file added for docker containers which will set the timezone to America/Chicago by default.

### CODE REVIEW

**Comment by medology-bugman** [comment]
Paging @goodevilgenius to review the back-end work in this pull request!

I couldn't recognize who these files were relevant to:
.gitignore
bin/composer
bin/containers
bin/php
circle.yml
docker/.env.example
docker/docker-compose.yml

**Comment by TaysirTayyab** [comment]
@goodevilgenius Need a re-review.

I made changes to the default timezone and I updated the `bin` scripts to wrap `$(pwd)` in quotes in case someone has the project root in a directory that has spaces.

File: bin/composer [hunk 1]
```diff
@@ -1,3 +1,3 @@
 #!/bin/bash +x
 
-docker run --rm -v $(pwd):/data -w /data composer/composer:1.1 $@
+docker run --rm --env-file "$(pwd)"/docker/.env -v $(pwd):/data -w /data composer/composer:1.1 $@
```
Comment (goodevilgenius) [inline bin/composer]: You'll need to enquote both `$(pwd)`s.

- [ ] Done
- [ ] Approved

File: bin/php [hunk 2]
```diff
@@ -1,6 +1,7 @@
 #!/bin/bash +x
 
-docker run --rm -it -v $(pwd):/app -w /app \
+docker run --rm -it --env-file "$(pwd)"/docker/.env \
+    -v $(pwd):/app -w /app \
```
Comment (goodevilgenius) [inline bin/php]: Quotes here too.

- [ ] Done
- [ ] Approved

**Pull request merged**